main.py

```diff
@@ -32,6 +47,7 @@ def reset(self):
 			'hdd_locked': True,
 			'dvd_present': True,
 			'dvd_path': '/path/to/disc.iso',
+			'iso_path': '/path/to/iso_folder',
```

Comment:
We not only support ISOs, I'd say `drive_path` or `disc_path` or something.
We can also get rid of the `dvd_path` at the same time probably?